src/tests/cli_tests.py

```diff
@@ -903,14 +928,25 @@ def test_sym_encryption__rnp_to_gpg(self):
             rnp_sym_encryption_rnp_to_gpg(cipher, size, 6, 'bzip2')
 
     def test_sym_encryption__rnp_aead(self):
+        AEAD_CIPHERS = ['AES', 'AES192', 'AES256', 'TWOFISH', 'CAMELLIA128', 'CAMELLIA192', 'CAMELLIA256']
+        AEAD_MODES = ['eax', 'ocb']
+        AEAD_BITS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 18, 24, 30, 40, 50, 56]
+
+        usegpg = gpg_supports_aead()
```

Comment:
I wonder if we need this. Shouldn't we just always have version which supports OCB and EAX?
If we use version which doesn't support those modes then test will simply pass without testing them? Doesn't sound good.

Reply:
If there is no latest gpg beta available then it will just do self-test. We should expand Travis to run with gpg beta, but this I think should be made as separate PR.